Application/Commands/RegisterSelectedCourseForApplicantCommand.cs

```diff
@@ -0,0 +1,56 @@
+﻿
+using Domain.Aggreagtes.ApplicantAggregate;
+using Domain.Aggreagtes.CourseAggregate;
+using Domain.Enums;
+using Domain.Repositories;
+using MediatR;
+
+
+namespace Application.Commands
+{
+    public class RegisterSelectedCourseForApplicantCommand : IRequest<ApplicantEnrollmentResponse>
+    {
+        public Guid ApplicantId { get; init; }
+        public Guid CourseId { get; init; }
+        public Guid CourseAssessmentId { get; init; }
+        public CourseMode CourseMode { get; init; }
+        public AssessmentMode AssessmentMode { get; init; }
+        public DateTime SubscriptionDate { get; init; }
```

Comment:
I don't understand this command. Can you give add notes to explain the purpose of this class and its members?

Reply:
The task is to implement "Endpoint to register a selected course"

when an applicant clicks on a course , he should be able to add the course.  going through the codebase, the only link between the two objects id ApplicationEnrolllment, which has a constructor that takes  all the parameters declared .